### ISSUE TITLE
Improve consistency with controlled vocabularies

### ISSUE THREAD

**Issue by tcarval** [nvs-vocabs/ArgoVocabs#44]
These variables may adopt controlled vocabularies

DEPLOYMENT_PLATFORM (https://github.com/nvs-vocabs/ArgoVocabs/issues/2)
DEPLOYMENT_CRUISE_ID (https://github.com/nvs-vocabs/ArgoVocabs/issues/3) 
DEPLOYMENT_REFERENCE_STATION_ID
PROJECT_NAME (https://github.com/nvs-vocabs/ArgoVocabs/issues/5) 
PI_NAME (https://github.com/nvs-vocabs/ArgoVocabs/issues/6) 
Accepted, should be restricted to pi_name variable
FLOAT_OWNER (https://github.com/nvs-vocabs/ArgoVocabs/issues/7) 
SPECIAL_FEATURES (https://github.com/nvs-vocabs/ArgoVocabs/discussions/24)
CUSTOMIZATION (https://github.com/nvs-vocabs/ArgoVocabs/discussions/24)
ANOMALY (https://github.com/nvs-vocabs/ArgoVocabs/discussions/24)
Pre-deployment information
Calibration information
LAT/LON - define coordinate reference system
SENSOR_MODEL 
https://github.com/nvs-vocabs/R27/issues/8
https://github.com/nvs-vocabs/R27/issues/13 
WMO_INST_TYPE (https://github.com/nvs-vocabs/ArgoVocabs/issues/35)
BATTERY_PACK, etc

**Comment by apswong** [comment]
@tcarval. Please list this:
- SENSOR_MODEL (Should sensor firmware version be moved to a new field? R27#8)
under https://github.com/nvs-vocabs/ArgoVocabs/issues

**Comment by vpaba** [comment]
Done now @apswong: https://github.com/nvs-vocabs/ArgoVocabs/issues/55

**Comment by delphinedobler** [comment]
@tcarval , could you refresh this issue ticket ? I'm not sure what actions are still needed. I am under the impression that this is a "hat" ticket.  Do we need to keep it open ? Or would you tag which vocabulary are not yet constrained (nor with its own opened ticket) ?

**Comment by tcarval** [comment]
Yes, this ticket contains links to other tickets. I prefer to close it as it does not explicitely manages actions.
